apps/minifront/src/components/shared/non-native-fee-warning.tsx

```diff
@@ -32,10 +94,18 @@ export const NonNativeFeeWarning = ({
    * />
    * ```
    */
-  hasStakingToken: boolean;
   wrap?: (children: ReactNode) => ReactNode;
 }) => {
-  const shouldRender = !!amount && !hasStakingToken;
+  const { data } = useGasPrices();
+  const stakingTokenMetadata = useStakingTokenMetadata();
```

Comment:
nit; can pass the `stakingTokenMetadata` field directly into `NonNativeFeeWarning` instead of fetching it again within the react component.

Reply:
Given they pull from the same cache, it isn't like it fetches more than once. But up to you!

Reply:
right, good point